specs/GetFavicon.md

```diff
@@ -0,0 +1,77 @@
+# Background
+A Favicon is an asset which is a part of every webpage, and typically displayed on each tab. Developers would
```

Comment:
A webpage doesn't necessarily have a favicon, correct? It would be good to also document what the value of the FaviconUrl property is when there's no favicon.